docs/node-support.md

```diff
@@ -6,7 +6,7 @@
 |------------------|------------------------------|
 | 0.10.x           | 0.0.0 - 2.8.x                |
 | 0.12.x           | 0.0.0 - 2.11.x               |
-| 4.x              | 1.13.9 - Current             |
+| 4.x              | 1.13.9 - 3.1.x               |
```

Comment:
unless this is retargeted to beta, this should be 3.2

Reply:
Yes, this was meant for beta. Next stable release will be after 4.x support is over, so it makes sense.